paymentsheet/res/layout/fragment_paymentsheet_payment_methods_list.xml

```diff
@@ -1,18 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can we also remove this root level linear layout?

Reply:
The next pull request will remove usage of this fragment altogether.